buildscripts/UploadTestresults.ps1

```diff
@@ -0,0 +1,8 @@
+﻿
+# upload results to AppVeyor
+$wc = New-Object 'System.Net.WebClient'
+
+$testBinPath = "bin\Release\net45"
+$wc.UploadFile("https://ci.appveyor.com/api/testresults/nunit/$($env:APPVEYOR_JOB_ID)", (Resolve-Path "src\Castle.Windsor.Tests\$testBinPath\TestResult.xml"))
+$wc.UploadFile("https://ci.appveyor.com/api/testresults/nunit/$($env:APPVEYOR_JOB_ID)", (Resolve-Path "src\Castle.Facilities.WcfIntegration.Tests\$testBinPath\TestResult.xml"))
```

Comment:
Thanks. We are obviously ignoring .NET Core for now so I'll also ignore the problem of what happens with tests with the same name until we get to that.

At first I thought your implementation seemed a bit like a hack but that appears to be the only documented way to get AppVeyor to read in a test results XML, the worker API is one test at a time.

Could you please move the PowerShell script into the appveyor.yml file, I'd like to keep the AppVeyor stuff contained.

I'll let you decide if you add it as a multi-line script or just as two single lines. It appears both ways are easily doable looking at these example configs:

```
on_success:
  - ps: |
      if ($true) {
        Write-Host "Success"
      }

install:
  - ps: (new-object net.webclient).DownloadFile('https://mysite.com/mypackage.msi', 'mypackage.msi')
  - ps: ...
```